make/RunTests.gmk

```diff
@@ -996,7 +996,8 @@ define SetupRunJtregTestBody
 	$$(RM) -r $$($1_TEST_RESULTS_DIR)
 
   $1_COMMAND_LINE := \
-      $$(JTREG_JAVA) $$($1_JTREG_LAUNCHER_OPTIONS) \
+      cd $$($1_TEST_SUPPORT_DIR) && $$(JTREG_JAVA) \
```

Comment:
Is TEST_SUPPORT_DIR guaranteed to be an absolute path?

Reply:
Yes.